### PR TITLE
Prefs [TASK][26319] Create Settings screen UI

### PR DESCRIPTION
**Pull request by toandv98**
## Related Tickets

- [#Task 26319](https://edu-redmine.sun-asterisk.vn/issues/26319)

## WHAT

- ###  Create Settings screen UI

## HOW

- ImageView display avata, Textview display name
- PreferenceScreen settings (API Token, Time sheet, Mention, Chat room, About)

## Review Checklist

| Category      | View Point                                                   | Description                                                  | Expected Reviewer Answer | Reviewer1 (ToanDV) | Reviewer2 (name)   |
| ------------- | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------ | ------------------ | ------------------ |
| General       | Code should be self-explanatory.                             | Get a feel of story reading, while going through the code. Use appropriate name for variables, functions and classes. If you are taking more time to understand the code, then either code needs refactoring or at least comments have to be written to make it clear. | YES                      | <li>- [x] yes</li> | <li>- [ ] yes</li> |
| General       | Is this function or class too big? If yes, is the function or class having too many responsibilities? |                                                              | NO                       | <li>- [x] no</li>  | <li>- [ ] no</li>  |
| General       | Does the code follow DRY?: Do not Repeat Yourself            | Is there any redundant or duplicate code? DRY (Do not Repeat Yourself) principle: The same code should not be repeated more than twice. Consider reusable services, functions and components. Consider generic functions and classes. | YES                      | <li>- [x] yes</li> | <li>- [ ] yes</li> |
| General       | Exception handling and cleanup (dispose) resources           |                                                              | YES                      | <li>- [x] yes</li> | <li>- [ ] yes</li> |
| General       | Does the code build and execute correctly?                   |                                                              | YES                      | <li>- [x] yes</li> | <li>- [ ] yes</li> |
| Conventions   | Does the source code have end-line errors?                   |                                                              | NO                       | <li>- [x] no</li>  | <li>- [ ] no</li>  |
| Conventions   | No hard coding, use constants/configuration values           |                                                              | YES                      | <li>- [x] yes</li> | <li>- [ ] yes</li> |
| Conventions   | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES                      | <li>- [x] yes</li> | <li>- [ ] yes</li> |
| Redmine       | Does the ticket follow Sun* Redmine working process?         | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md | YES                      | <li>- [x] yes</li> | <li>- [ ] yes</li> |
| Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |                                                              | YES                      | <li>- [x] yes</li> | <li>- [ ] yes</li> |

## Notes (Optional)

*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/xml/root_preferences.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (tuannm-1276) [inline app/src/main/res/xml/root_preferences.xml:1]: wrong convention naming in this file

File: app/src/main/res/values/arrays.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<resources>
+    <!-- Reply Preference -->
+    <string-array name="reply_entries">
+        <item>Reply</item>
+        <item>Reply to all</item>
```
Comment (aug16vcc) [inline app/src/main/res/values/arrays.xml]: Bỏ vào `strings.xml`

File: app/src/main/res/values/arrays.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    </string-array>
+
+    <string-array name="reply_values">
+        <item>reply</item>
+        <item>reply_all</item>
```
Comment (aug16vcc) [inline app/src/main/res/values/arrays.xml]: Bỏ vào `strings.xml`

**Review by tuannm-1276 — APPROVED** [review]

File: app/src/main/res/values/arrays.xml (hunk 3 above)
Comment (toandv98) [reply app/src/main/res/values/arrays.xml]: File này ko dùng đến nữa mà t quên xóa :]]

**Pull request merged**